kge/job/grid_search.py

```diff
@@ -68,8 +68,12 @@ def run(self):
 
         # and run it
         if self.config.get("grid_search.run"):
-            job = Job.create(self.config, self.dataset, parent_job=self)
-            job.resume()
+            job = JobFactory.load_from(
+                checkpoint_file=None,
+                config=self.config,
+                dataset=self.dataset,
```

Comment:
Where does the dataset come from?

Reply:
it is now created either, when the job is created or resumed